agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4349,15 +4349,61 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
         }
     }
 
-    if (cmdu_rx.getNextTlvType() != uint8_t(ieee1905_1::eTlvType::TLV_VENDOR_SPECIFIC)) {
-        LOG(ERROR) << "Not vendor specific TLV (not Intel?)";
-        return false;
-    }
+    if (cmdu_rx.getNextTlvType() == uint8_t(ieee1905_1::eTlvType::TLV_VENDOR_SPECIFIC)) {
```

Comment:
still - a few lines up the manufacturer check will return false for none intel controller.